applications/physical_systems/ising_model/ising_model.ipynb

```diff
@@ -29,12 +29,6 @@
    "execution_count": 1,
```

Comment:
<p>Missing capital letter in 'We'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>